tests/Feature/ApplicationTest.php

```
@@ -1224,4 +1260,352 @@ private function assertWorkerNotRestarted(int|string $workerId): void
+        $this->actingAs($this->user);
+
+        $this->put(route('application.update-env', [
+            'server' => $this->server,
+            'site' => $this->site,
+        ]), [
+            'env' => 'APP_NAME=Test',
+            'variables' => [
+                ['key' => 'APP_NAME', 'value' => 'Test', 'is_secret' => false],
+            ],
+        ])->assertSessionHasErrors('env');
+    }
```

PHPMD (2.15.0)

[error] The method test_update_env_rejects_a_submission_with_both_keys is not named in camelCase. (undefined)

(CamelCaseMethodName)